Windows_Setup.bat

```diff
@@ -0,0 +1,40 @@
+@echo off
+
+:: This batch script builds and installs libecl for Windows
+
+:: ENVIRONMENT AND PATH VARIABLES
+:: These path variables should point to the appropriate folders on your PC after installing Visual Studio. 
+:: This build requires a path to cli.exe (for your build (x64/32)), cmake.exe
+
+set PATH=%PATH%;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.15.26726\bin\Hostx64\x64
+set PATH=%PATH%;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.15.26726\include
+set PATH=%PATH%;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.15.26726\lib\x64
+set PATH=%PATH%;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin
+
+:: Set zlib, note Anaconda often comes with zlib so you may not need it yourself
+:: set PATH=%PATH%;C:\dev\zlib\bin;C:\dev\zlib\include;C:\dev\zlib\lib
+
+:: your prefered location for the libecl install 
+:: the lib\Release, bin\Release will need to be added to your PATH after
+:: the lib\python*\site-packages will need to be added to your PYTHONPATH
+set INSTALLPATH=C:\dev\ERT
+
+:: make build directory
+mkdir temp-build
+cd temp-build
+
+:: Codes for building ERT
+:: The cmake -G option made need to change if you have a newer/older version of Visual Studio
+cmake -G "Visual Studio 15 2017 Win64" -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=Release -DENABLE_PYTHON=ON -DERT_BUILD_CXX=OFF -DBUILD_APPLICATIONS=ON ..
+cmake --build . --config Release
+
+:: copy builds to install path
+XCOPY /Y lib\Release %INSTALLPATH%\lib\Release
+XCOPY /Y bin\Release %INSTALLPATH%\bin\Release
+cd lib
+
+FOR /d %%a in (python*) DO XCOPY %%a %INSTALLPATH%\lib\%%a /s /e /h /y
+
+cd ..\..
```

Comment:
This last part confuses me a bit. I would expect users to run `nmake install` (or `ninja install` if they want things a bit faster). If that doesn't work, I'd say we have a bug and we should fix it, copying files directly seems to be a workaround

Reply:
You can set the install path for the binary builds but the ?PYTHONPREFIX? option does not work under Windows so the ecl site-packages folder ends up in an obscure part of the build folder structure. I didn't experiment with _nmake_ or _ninja_, would these resolve that issue?

We could also resolve most of these comments by removing the bat file. It sounds like I wasn't using the best approach to building **libecl** under windows but I struggled to get other methods to work. Sorry for my ignorance with c compilers.

Reply:
I may be wrong `nmake` and `ninja` would perform pretty much the same operations as building the `INSTALL` target in visual studio, so I guess this might actually be a bug in libecl. I'll try to reproduce it and open a dedicated issue if needed

Reply:
Apologies, I must have been doing something wrong. I got the VS generator to build and install correctly. I've updated the example in the markdown file to reflect a VS example. Hopefully this rounds us out.